samples/bluetooth/mesh/light/README.rst

```diff
@@ -30,6 +32,27 @@ The sample also requires a smartphone with Nordic Semiconductor's nRF Mesh mobil
 
 .. include:: /includes/tfm.txt
 
+DFU requirements
+================
+
+The configuration overlay :file:`overlay-dfu.conf` enables DFU support in the application, and applies for the nRF52 series, including:
+
+* nrf52dk_nrf52832
+
+* nrf52840dk_nrf52840
+
+* nrf52833dk_nrf52833
+
+While this overlay configuration is only applicable for the nRF52 series in this sample, DFU over SMP can be utilized on other platforms as well.
+
+.. note::
+   Point-to-point DFU for :ref:`zephyr:thingy53_nrf5340` is supported by default.
```

Comment:
```suggestion
   Point-to-point DFU for :ref:`zephyr:thingy53_nrf5340` is supported by default.
   See :ref:`thingy53_app_update` for more information about updating firmware image on :ref:`zephyr:thingy53_nrf5340`.
```
I wonder if it is better to link to https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-9109/nrf/ug_thingy53.html#updating-firmware-image than to https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-9109/nrf/ug_thingy53_gs.html#getting-started-with-precompiled-firmware-samples. What do you think?